### PR TITLE
[add] Transaction Buy/View tabs

### PR DESCRIPTION
**Pull request by vtumrukota**
Also updated the routes to implement nested views on the first transaction page

### CODE REVIEW

**Comment by josephlin55555** [comment]
looks pretty good

File: snapmarketapp/app/styles/main.css [hunk 1]
```diff
@@ -1 +1,5 @@
-/* Your application styles go here */
\ No newline at end of file
+/* Your application styles go here */
+
+.transaction {
```
Comment (andyttran) [inline snapmarketapp/app/styles/main.css]: is this being applied or style.css?

File: snapmarketapp/app/templates/buy-offers.html [hunk 2]
```diff
@@ -1,14 +1,3 @@
...
-  'friend' is a $scope variable created in the FriendsCtrl controller (controllers.js)
-  The FriendsCtrl pulls data from the Friends service (service.js)
-  The Friends service returns an array of friend data
--->
-<ion-view view-title="{{chat.name}}">
-  <ion-content class="padding">
-    <img ng-src="{{chat.face}}" style="width: 64px; height: 64px">
     <p>
-      {{chat.lastText}}
+     BUY MAD SHIT
```
Comment (andyttran) [inline snapmarketapp/app/templates/buy-offers.html]: word.

**Comment by andyttran** [comment]
Looks good. Ill merge

**Pull request merged**